libarchive/libarchive-3.3.2-vita.patch

```diff
@@ -22,9 +22,32 @@ diff -Naur libarchive-3.3.2.orig/libarchive/archive_read_disk_posix.c libarchive
  #endif
  #ifndef O_BINARY
  #define O_BINARY	0
+diff -Naur libarchive-3.3.2.orig/libarchive/archive_read_support_filter_program.c libarchive-3.3.2/libarchive/archive_read_support_filter_program.c
+--- libarchive-3.3.2.orig/libarchive/archive_read_support_filter_program.c	2017-07-10 04:38:04.000000000 +0200
++++ libarchive-3.3.2/libarchive/archive_read_support_filter_program.c	2022-05-12 12:27:32.993098967 +0200
+@@ -58,7 +58,13 @@
+ #include "archive_read_private.h"
+ #include "filter_fork.h"
+ 
+-
++#if 1
```

Comment:
```suggestion
+#ifdef __vita__
```

Reply:
Hi, i'm actually porting this to multiple platforms, it's the reason of this. Is it really useful to add a  specific define here ?

Reply:
Well, if you ever want to upstream those changes - yes

Reply:
I have to redo the whole patch then,  because it's full of this :)